velox/core/QueryCtx.cpp

```diff
@@ -19,23 +19,43 @@
 
 namespace facebook::velox::core {
 
-QueryCtx::QueryCtx(
+/*static*/ std::shared_ptr<QueryCtx> QueryCtx::create(
```

Comment:
// static
on top?

Reply:
It looks like we write /*static*/ in the same line with function as the other places in our code base.